cardano-db-sync-extended/src/Cardano/DbSync/Plugin/Extended.hs

```diff
@@ -19,7 +19,9 @@ extendedDbSyncNodePlugin backend =
               ++ [epochPluginOnStartup backend]
         , plugInsertBlock =
             plugInsertBlock defPlugin
-                ++ [epochPluginInsertBlock backend]
```

Comment:
Does not seem correct for that to be removed.

Reply:
It is moved to `plugInsertBlockDetails`